tests/components/homekit/test_type_covers.py

```diff
@@ -151,6 +151,12 @@ def cls():
     assert acc.char_target_position.value == 60
     assert acc.char_position_state.value == 1
 
+    hass.states.async_set(entity_id, STATE_OPENING, {ATTR_CURRENT_POSITION: 70.0})
+    await hass.async_block_till_done()
+    assert acc.char_current_position.value == 70
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4876356)